src/settings/_default.settings

```diff
@@ -84,6 +84,13 @@
     "category": "Qt",
     "setting": "window_geometry_v2"
   },
+  {
+    "value": "False",
```

Comment:
The value should just be `false`, lowercase with no quotes. That'll store it as a boolean value instead of a string. Otherwise, setting it to `True` (without quotes) later will change the type.

Reply:
Agree, value should be `false`, for JSON sake